lib/index.js

```diff
@@ -20,15 +22,30 @@ const LICENSES = [
   'licence.md'
 ]
 
+const noop = () => {}
+
 module.exports = getLicense
 function getLicense (modulePath, optionalCallback) {
+  const cb = optionalCallback || noop
```

Comment:
Instead of `if-else` checks everywhere we can just default to `noop` and use that all over the place for the callback interface.